src/components/comment-form/comment-form.twig

```diff
@@ -12,13 +22,16 @@
     >
       {{ heading_text|default("Leave a Comment") }}
     </{{_heading_tag}}>
-    <p {% if help_text_id %}id="{{ help_text_id }}"{% endif %}>
-      Please be kind, courteous and constructive.
-      You may use simple HTML or
-      <a href="https://en.support.wordpress.com/markdown-quick-reference">Markdown</a>
-      in your comments.
-      All fields are required.
-    </p>
+
+    {% if wrap_help_text_in_alert %}
+      {% embed '@cloudfour/components/alert/alert.twig' %}
+        {% block content %}
+          {{help_text}}
+        {% endblock %}
+      {% endembed %}
+    {% else %}
+      {{help_text}}
+    {% endif %}
```

Comment:
Would this be simpler if we used blocks instead of a property?

```suggestion
    {% block help_text %}
      {% if help_text is not empty %}
        {% embed '@cloudfour/components/alert/alert.twig' %}
          {% block content %}
            {{help_text}}
          {% endblock %}
        {% endembed %}
      {% endif %}
    {% endblock %}
```

Reply:
I'm not sure. This component is used in two different places:

- Leaving a comment on a post
- Responding to a comment on a post

I think the following things are the same in both use cases:

- The help text content
- The `aria-describedby` link between the help text and the `textarea`

I think the following is different:

- Whether the help text should be wrapped in an alert.

If I'm understanding your suggestion correctly, we would need to pass in the help text content, and maintain the `aria-describedby` link manually when we didn't want the help text in an alert. That seems like more of a maintenance burden than adding a new property, but I may be misunderstanding the suggestion.

Reply:
When do we not want the help text in an alert?

Reply:
Ahh, perhaps I had misunderstood. My impression was we didn't want to use the alert in the main "Leave a comment" form. 

![image](https://user-images.githubusercontent.com/5798536/126708956-b1098228-76f6-408f-a37c-f3c56bae9785.png)

If that's not the case then you're right that this new prop is over-complicating things. I can switch to using an alert everywhere.

Reply:
Yeah, let's keep it consistent!